spec/controllers/games_controller_spec.rb

```diff
@@ -2,13 +2,23 @@
 require 'pry'
 
 RSpec.describe GamesController, type: :controller do
+  describe 'games#new action' do
+    it 'should login players' do
+      player = FactoryGirl.build(:player)
+      sign_in player
+    end
+
+    it 'should initialize pieces on the board' do
+      @game = FactoryGirl.build(:game)
+      @piece = FactoryGirl.build(:piece)
+    end
```

Comment:
These aren't testing anything yet. Take a look here for examples:
https://www.relishapp.com/rspec/rspec-rails/docs/controller-specs